stl/src/ios.cpp

```diff
@@ -31,7 +31,10 @@ void __CLRCALL_PURE_OR_CDECL ios_base::_Ios_base_dtor(ios_base* _This) { // dest
     }
 
     _This->_Tidy();
-    delete _This->_Ploc;
+
+    if (_This->_Ploc != nullptr) {
+        delete _This->_Ploc;
+    }

```

Comment:
I don't think that this change is necessary - `delete` is a guaranteed no-op for null. See WG21-N4868
17.6.3.2 [new.delete.single]/17 "If `ptr` is null, does nothing."

Reply:
Honestly, I didn't know this point. Thank you very much for letting me know about this guarantee.

Reply:
You're welcome! :smile_cat: This requirement is also binding on any user-replacement implementations of `operator delete` so we don't need to worry about them.